common/capabilities/channel.go

```diff
@@ -28,6 +28,9 @@ const (
 
 	// ChannelV2_0 is the capabilities string for standard new non-backwards compatible fabric v2.0 channel capabilities.
 	ChannelV2_0 = "V2_0"
+
+	// ChannelV2_5 is the capabilities string for standard new non-backwards compatible fabric v2.5 channel capabilities.
+	ChannelV2_5 = "V2_5"
```

Comment:
"Channel" capabilities are capabilities that impact both orderer nodes and peer nodes.
"Application" capabilities are capabilities that impact only peer nodes.

Since the PurgePvtData capability only impacts peers, it should be an "Application" capability only.
You can remove this change and all the subsequent changes related to "Channel" capabilities.
The "Application" capabilities changes above look correct.